code/assurance/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.assurance"
+    compose = true
+    //enableSpotless = true
```

Comment:
Disabled spotless for this PR. Will enable this in the next PR to make the build pass.